main/release/utils/aws_arn_helper.py

```diff
@@ -0,0 +1,17 @@
+from boto3 import Session
+
+
+class ArnTopicNotFound(Exception):
+    pass
+
+
+class AwsArnHelper:
+
+    @staticmethod
+    def get_ssm_arn_topic_name(aws_session: Session, parameter_name: str) -> str:
```

Comment:
This method basically gets the value of `parameter_name` ssm parameter. What do you think about using a more direct method name? For example: `get_ssm_parameter_value`

Reply:
I renamed it, seems more readable to you that way. (I don't have a strong opinion about it personally.)